.github/workflows/trivy-scan.yml

```diff
@@ -0,0 +1,77 @@
+name: Trivy CVE Dependency Scanner
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+  workflow_dispatch:
+
+jobs:
+  scan-image:
+    runs-on: ubuntu-latest
+    permissions:
+      security-events: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Set up Go 1.x
+        uses: actions/setup-go@v1
+        with:
+          go-version: "1.17"
+      - name: Install Carvel Tools
+        uses: vmware-tanzu/carvel-setup-action@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          only: ytt, kapp, kbld, vendir
+          ytt: v0.37.0
+          kapp: v0.42.0
+          kbld: v0.31.0
+          vendir: v0.23.0
+      - name: Build image from Dockerfile
```

Comment:
I don't think we need this, right? Looks like all that is happening here is a `docker build` and trivy-related things.

Reply:
Right, I realized I am missing a commit which tests the kapp-controller binary in addition to the docker image. That's why I had it in there. I am going to pull this PR back to draft while I add that in and then re-open it up